src/algolia-queries.js

```diff
@@ -0,0 +1,47 @@
+const escapeStringRegexp = require('escape-string-regexp');
+const { default: slugify } = require('slugify');
+
+const pagePath = `pages/`; //path to the docs pages /pages/docs/ect...
+const indexName = `Pages`; //Algolia querie index name
+
+const pageQuery = `{
+  pages: allMdx(
+    filter: {
+      fileAbsolutePath: { regex: "/${escapeStringRegexp(pagePath)}/" },
+    }
+  ) {
+    edges {
+      node {
+        id
+        frontmatter {
+          title
+        }
+        excerpt(pruneLength: 3000)
```

Comment:
Does this mean we'll only be able to search the first 3k characters of each page rather than the full `body`? I see they have a 10KB limit for index items, which is a bummer. But, maybe we can at least make this a little higher since we don't have much else in the index? 8k?